packages/config/package.json

```diff
@@ -14,7 +14,8 @@
     "test:compilation": "tsc --noEmit",
     "build": "tsc && copyfiles 'src/**/*.json' build/dist",
     "build:clean": "rm -rf build",
-    "extract:translations": "bash extract-translations.sh"
+    "extract:translations": "bash extract-translations.sh",
+    "debug": "../../debug-service-in-chrome.sh 2021"
```

Comment:
Don't have a fix suggestion... but I think the packages themselves cannot refer to the root folder, as the packages are self-contained and could be published as-is. In development this could not matter though.

Still worth considering if there's a way to refactor this 🤔